### PR TITLE
Introduce MockServerClient

### PR DESCRIPTION
**Pull request by lindycoder**
Fully functional with tests and a good README

Let's make MockServer testing in python friendly!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+[![Build Status](https://travis-ci.org/internap/python-mockserver-client.svg?branch=master)](https://travis-ci.org/internap/python-mockserver-client)
+
+Python client to James D. Bloom's awesome MockServer : http://www.mock-server.com/
+
+## Philosophy
+
+Tests should be readable and Mock Server is very complete and customizable.  This library tries to keep it simple and
+straight forward using python's kwargs to avoid big declarations.
+
+We think a mock should be able to fit one 120 char line of code should the expectation be simple enough :)
```
Comment (pmillette) [inline README.md]: fit into one

File: README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+# MockServer Python Client
+
+[![Build Status](https://travis-ci.org/internap/python-mockserver-client.svg?branch=master)](https://travis-ci.org/internap/python-mockserver-client)
+
+Python client to James D. Bloom's awesome MockServer : http://www.mock-server.com/
+
+## Philosophy
+
+Tests should be readable and Mock Server is very complete and customizable.  This library tries to keep it simple and
+straight forward using python's kwargs to avoid big declarations.
```
Comment (pmillette) [inline README.md]: straight-forward

File: README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
+# MockServer Python Client
+
+[![Build Status](https://travis-ci.org/internap/python-mockserver-client.svg?branch=master)](https://travis-ci.org/internap/python-mockserver-client)
+
+Python client to James D. Bloom's awesome MockServer : http://www.mock-server.com/
+
+## Philosophy
+
+Tests should be readable and Mock Server is very complete and customizable.  This library tries to keep it simple and
```
Comment (pmillette) [inline README.md]: is already very complete

File: README.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+client = MockServerClient("http://localhost:1080")
+
+client.stub(
+    request(method="GET", path="/that/thing", querystring={"is": "good"}, headers={"so": "good"}),
+    response(code=418, body="i'm a teapot", headers={"hi": "haa"})
+)
+```
+
+* You can also add a `times(N)` as third parameters to limit how many times this stub can be called, default is infinite
```
Comment (pmillette) [inline README.md]: as a third parameter

File: README.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+client = MockServerClient("http://localhost:1080")
+
+client.reset()
+```
+
+## Customizing and shortcuts
+
+This client consumes Mock Server's REST API : https://app.swaggerhub.com/apis/jamesdbloom/mock-server-openapi
+
+Here are shortcuts already ready, there may be more to come
```
Comment (pmillette) [inline README.md]: Here's a few shortcuts

File: README.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+    request(path="/stuff"),
+    json_response(code=200, body={"full": "json", "structure": ["with", "stuff", 1]})
+)
+```
+
+This automatically dumps the body in json and appends the `application/json` Content-Type to the headers.
+
+## More documentation
+
+There is currently no official documentation, you can look at the tests they are pretty explicit and simple to follow.
```
Comment (pmillette) [inline README.md]: suggestion "There is currently no official documentation, however you can consider the tests as a type of documentation, they are pretty explicit and simple to follow and help to clarify the purpose of each feature"

File: README.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+This automatically dumps the body in json and appends the `application/json` Content-Type to the headers.
+
+## More documentation
+
+There is currently no official documentation, however you can consider the tests as a type of documentation, they are
+pretty explicit and simple to follow and help to clarify the purpose of each feature.
+
+## Good practices
+
+Having your test setup/teardown like this is a probably a good idea.
```
Comment (simon-begin) [inline README.md]: like this is ~~a~~ probably a good idea

Comment (lindycoder) [reply README.md]: fixed

**Review by simon-begin — APPROVED** [review]

**Pull request merged**